src/templates/test.tsx

```diff
@@ -41,6 +42,6 @@ const Test = (props: any) => {
 };
 
 export const render = (data: any) =>
-  reactWrapper(data, 'Product Test', 'test.tsx', renderToString(<Test data={data} />), true);
+  reactWrapper(data, 'Product Test', 'test.tsx', renderToString(<Test data={data} />), true, SchemaWrapper(data));
```

Comment:
There was no change to this function so I'm assuming you forgot to commit something.